frontends/election-manager/src/screens/logic_and_accuracy_screen.tsx

```diff
@@ -13,25 +13,43 @@ export function LogicAndAccuracyScreen(): JSX.Element {
   return (
     <NavigationScreen>
       <Prose maxWidth={false}>
-        <h1>L&amp;A Materials</h1>
+        <h1>L&A Testing Documents</h1>
         {isLiveMode ? (
           <p>
-            L&amp;A materials are not available after live CVRs have been
-            imported.
+            L&A materials are not available after live CVRs have been imported.
           </p>
         ) : (
           <React.Fragment>
             <p>
-              Go to the “Tally” tab and print the “Unofficial Full Election
-              Tally Report”. This report is referred to as the “Zero Report”
-              because — before CVRs have been imported — all tallies should be
-              zero.
+              Print the following reports and ballots in preparation for L&A
+              Testing.
+            </p>
+            <h2>1. Unofficial Full Election Tally Report</h2>
+            <p>
+              Print this tally report to document that no votes have been
+              tallied before L&A testing begins.
+            </p>
+            <p>
+              <em>
+                Go to the Reports tab to print the Unofficial Full Election
+                Tally Report at the top of the page.
+              </em>
+            </p>
+            <h2>2. Precinct L&A Packages</h2>
+            <p>
+              Each Precinct L&A Package contains marked ballots, unmarked
+              ballots, and a tally report with expected results.
             </p>
             <p>
               <LinkButton to={routerPaths.testDecks}>
-                Print L&amp;A Packages
+                List Precinct L&A Packages
               </LinkButton>
             </p>
+            <h2>3. Test Deck Tally Report for All Precincts</h2>
+            <p>
+              This report has the results that are expected after scanning all
+              the ballots.
+            </p>
```

Comment:
I learned during the L&A project that the tally report includes double the counts of all the ballots since we scan everything on a precinct scanner and a central scanner (@adghayes taught me this!). Do we need to mention that anywhere?

This page is already getting quite verbose, so I understand if that doesn't need to be mentioned or is implied

Reply:
I know that this was an issue in the past, however I'm not sure this is still the case. 

@ginvdr - Can you clarify?

Reply:
I don't think we need to add instruction about that @arsalansufi but it's a good question to raise.  I'll make sure it's in our documentation and covered in training.

Reply:
Awesome, I figured as much!